entries/resizable.xml

```diff
@@ -153,12 +153,24 @@
 					<desc>The size represented as <code>{ width, height }</code> before the resizable is resized</desc>
 				</property>
 				<property name="position" type="Object">
-					<desc>The current position represented as <code>{ left, top }</code></desc>
+					<desc>The current position represented as <code>{ left, top }</code>. The values may be changed to modify where the element will be positioned. Useful for custom resizing logic.</desc>
 				</property>
 				<property name="size" type="Object">
-					<desc>The current size represented as <code>{ width, height }</code></desc>
+					<desc>The current size represented as <code>{ width, height }</code>. The values may be changed to modify where the element will be positioned. Useful for custom resizing logic.</desc>
 				</property>
 			</argument>
+			<example>
+				<desc>Constrain horizontal resizing:</desc>
```

Comment:
This does the same thing as `maxWidth`, which isn't that great as an example. Here's a callback that I've been using in an application, restricting the resizable to a vertical grid of 30px:

```js
resize: function(event, ui) {
    ui.size.height -= ui.size.height % 30;
    ui.size.height += ui.originalPosition.top % 30;
}
```

Maybe add this in addition to the current example?